index.html

```diff
@@ -85,88 +90,200 @@
+          effect of the request MUST NOT be durable unless and until the series
+          is successfully <a href="#commit">finished</a>.
+        </p>
+        <p>
+          If the request is successful, the server MUST include a
+          <code>Atomic-ID</code> response header whose value MUST be the current
+          transaction identifier.
+        </p>
+        <p>
+          If the transaction identified by the <code>Atomic-ID</code> header is
+          invalid for any reason, the server MUST NOT update any resources, and
+          it MUST respond with a <code>409 Conflict</code> HTTP status. The
+          server MAY also include more information about the reason for
+          rejecting the request in the response body.
         </p>
       </section>
 
       <section id="expiration">
         <h2>Expiration</h2>
         <p>
-          An implementation MAY enforce automatic expiration for an atomic series of requests.
-          If so, the response to any request made as part of the series MUST have a
-          <code>Atomic-Expires</code> header with the HTTP datetime at which the series will
-          expire. An implementation MAY extend the lifetime of a series upon the receipt of any
-          request specifying the unique identifier of that series in the <code>Atomic-ID</code>
-          header.
+          An implementation MAY enforce automatic expiration for an atomic
+          series of requests.  If so, the response to any request made as part
+          of the series MUST have a <code>Atomic-Expires</code> header with the
+          HTTP datetime at which the series will expire.
         </p>
+        <p>
+          An implementation SHOULD extend the lifetime of a series upon the
+          receipt of any request to resources within the scope of the
+          transaction.
+        </p>
+        <p>
+          An implementation SHOULD permit extending the lifetime of a series by
+          a <code>POST</code> request to the transaction identifier URI.
       </section>
 
       <section id="commit">
         <h2>Committing a series</h2>
         <p>
-          A client can finish an atomic series of requests by sending any
-          <code>PUT</code>/<code>POST</code>/<code>PATCH</code>/<code>DELETE</code> request
-          including both a <code>Atomic-Commit</code> header and a <code>Atomic-ID</code>
-          header with the unique identifier of an existing series. The effects of all requests
-          in the series MUST become durable and MUST become visible to all clients. The unique
-          identifier of the series MUST be expired and MUST NOT be reused.
+          A client finishes an atomic series of requests by sending a
+          <code>PUT</code> request to the transaction identifier URI.
+        </p>
+        <p>
+          The request body of this request MAY be non-empty. If it is non-empty,
+          [TODO: should we specify? or leave it up to implementations; this
```

Comment:
We're not necessarily linking transactions to versions, so commit message wouldn't necessarily have anywhere to go. My inclination is to leave it up to the implementation what could go in the request body.

Reply:
I agree.